msgraph-mail/php/src/Models/Microsoft/Graph/DateTimeTimeZone.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Microsoft\Graph\Models\Microsoft\Graph;
+
+use Microsoft\Kiota\Abstractions\Serialization\Parsable;
+use Microsoft\Kiota\Abstractions\Serialization\ParseNode;
+use Microsoft\Kiota\Abstractions\Serialization\SerializationWriter;
+
+class DateTimeTimeZone implements Parsable 
+{
```

Comment:
this is missing a constructor to initialize additional data
also, I'm assuming php classes are public by default

Reply:
Yes classes are public.